datarobot_batch_scoring/batch_scoring.py

```diff
@@ -134,7 +137,7 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                     [c for c in keep_cols if c not in first_row.fieldnames],
                     first_row.fieldnames))
 
-        if not dry_run:
+        if not (dry_run or import_id):
```

Comment:
Reaaly this authorize request check is serving as early check for validity of model/project/incoming data as it's done with first row of incoming file before starting threads and so on, so it fails much earlier.

So really we need to do it for SSE as well.